### PR TITLE
First commit for piler

### PR DESCRIPTION
**Pull request by oinizan**
* [x] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).
This PR add a new recipe so I mention @bioconda/core to get a review.

### CODE REVIEW

File: recipes/piler/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+package:
+  name: piler
+  version: "0.1"
+
+build:
+  number: 0
+  skip: True # [osx]
```
Comment (bgruening) [inline recipes/piler/meta.yaml]: Is this really needed. Would be great to compile it an OS-X as well :)

File: recipes/piler/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+source:
+  fn:  piler_source.tar.gz
+  url: http://www.drive5.com/piler/piler_source.tar.gz
+  md5: 6f351fb437d0cc6a1ec84f41998621c9
+
+requirements:
+  build:
+  - gcc
```
Comment (bgruening) [inline recipes/piler/meta.yaml]: - gcc # [linux]
- llvm # [osx]

File: recipes/piler/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  run:
+  - libgcc
```
Comment (bgruening) [inline recipes/piler/meta.yaml]: - libgcc # [linux]

**Comment by oinizan** [comment]
Ok for osx compatibility

**Pull request closed without merging**